src/mut/writers.cpp

```diff
@@ -75,6 +82,9 @@ void swc(const Morphology& morphology, const std::string& filename)
         return;
     }
 
+    if (hasPerimeterData(morphology))
```

Comment:
IMO it is more readable and less error-prone to put braces even around statements with one instruction.
Besides, there is a [clang-tidy check for that](https://clang.llvm.org/extra/clang-tidy/checks/readability-braces-around-statements.html) I recommend.